lib/graphql/schema/relay_classic_mutation.rb

```diff
@@ -33,9 +33,10 @@ def resolve_with_support(**inputs)
         # But when using the interpreter, no instrumenters are applied.
         if context.interpreter?
           input = inputs[:input].to_kwargs
+
           # Transfer these from the top-level hash to the
           # shortcutted `input:` object
-          self.class.extras.each do |ext|
+          self.class.extras(field&.extras).each do |ext|
```

Comment:
I think calling `.extras(...)` will [clobber any class-defined extras](https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/resolver.rb#L209), which could be unexpected. How about _merging_ these values instead, for example: 

```suggestion
          all_extras = self.class.extras + field.extras 
          all_extras.each do |ext|
```

?

Reply:
@rmosolgo great point! I didn't see that. Fixed now 561241f01
Thanks